app/uk/gov/hmrc/gform/validation/ComponentValidator.scala

```diff
@@ -654,6 +673,165 @@ object ComponentValidator {
         )
     }
 
+  private def validateSterling(
+    fieldValue: FormComponent,
+    value: String,
+    isPositive: Boolean,
+    isWhole: Boolean
+  )(implicit
+    messages: Messages,
+    sse: SmartStringEvaluator
+  ): ValidatedType[Unit] = {
+    val maxWhole = ValidationValues.sterlingLength
+    val errorExample = fieldValue.errorExample.getOrElse(SmartString.blank)
```

Comment:
I think if would be worth to make `errorExampleOrBlank` property of `FormComponent` and use that instead, rather than passing it around.

Reply:
To be honest, I'm uncertain about that. So far, I've come across situations where, for instance, if errorShortNameStart is defined, we use one message template, and if it's not, we use a different one. I can envision a similar scenario for errorExample. If we set it to be blank by default, we won't distinguish it from the cases when a designer made it explicitly to be blank.

Reply:
To clarify I'm talking about adding this to the `FormComponent`
```
val errorExampleOrBlank = errorExample.getOrElse(SmartString.blank) 
```
It is just a convenience value to be used when needed.